src/types.ts

```diff
@@ -1,7 +1,21 @@
 import { MiddlewareAPI, Dispatch, AnyAction, Action } from 'redux'
+import { AuthorizationState } from './modules/authorization/reducer'
+import { TransactionState } from './modules/transaction/reducer'
+import { WalletState } from './modules/wallet/reducer'
+import { FeaturesState } from './modules/features/reducer'
+import { LoadingState } from './modules/loading/reducer'
 
 export type RootDispatch<A extends Action = AnyAction> = Dispatch<A>
 
 export type RootMiddleware = (
   store: MiddlewareAPI<any>
 ) => (next: RootDispatch<AnyAction>) => (action: AnyAction) => any
+
+export type RootState = {
```

Comment:
We can't ensure that the dApps will have this root state, as any of them can load the modules they want. We might need to revert that back.

Reply:
sure, but RootStateOrAny was removed in react-redux v8, so I'l make it as any